src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -29,52 +29,43 @@
 namespace PrestaShopBundle\Controller\Admin\Sell\Catalog;
 
 use Exception;
-use Feature;
 use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureConstraintException;
 use PrestaShop\PrestaShop\Core\Domain\Feature\Exception\FeatureNotFoundException;
 use PrestaShop\PrestaShop\Core\Domain\Feature\Query\GetFeatureForEditing;
-use PrestaShopBundle\Bridge\AdminController\ControllerConfiguration;
-use PrestaShopBundle\Bridge\AdminController\FrameworkBridgeControllerInterface;
-use PrestaShopBundle\Bridge\AdminController\FrameworkBridgeControllerListTrait;
-use PrestaShopBundle\Bridge\AdminController\FrameworkBridgeControllerTrait;
-use PrestaShopBundle\Bridge\Helper\Listing\HelperBridge\FeatureHelperListBridge;
-use PrestaShopBundle\Bridge\Helper\Listing\HelperListConfiguration;
-use PrestaShopBundle\Bridge\Smarty\FrameworkControllerSmartyTrait;
+use PrestaShop\PrestaShop\Core\Domain\ShowcaseCard\Query\GetShowcaseCardIsClosed;
+use PrestaShop\PrestaShop\Core\Domain\ShowcaseCard\ValueObject\ShowcaseCard;
+use PrestaShop\PrestaShop\Core\Search\Filters\FeatureFilters;
 use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
 use PrestaShopBundle\Security\Annotation\AdminSecurity;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Tools;
 
 /**
  * Controller responsible for "Sell > Catalog > Attributes & Features > Features" page
  */
-class FeatureController extends FrameworkBundleAdminController implements FrameworkBridgeControllerInterface
+class FeatureController extends FrameworkBundleAdminController
 {
-    use FrameworkBridgeControllerTrait;
-    use FrameworkBridgeControllerListTrait;
-    use FrameworkControllerSmartyTrait;
-
     /**
      * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
      */
-    public function indexAction(Request $request): Response
```

Comment:
Can you please leave this old horizontal action code for now, we're probably gonna need some elements from it when we'll handle removing the `legacyLayout` It'd be easier if we know where to find the code instead of going back to the history.

You can rename it to `legacyHorizontalIndexAction` it doesn't need to have a related route, and you can add a comment that explains why we keep it for now (until legacyLayout is refactored) and that it should be cleaned after it's done

Reply:
hmm are you sure? We can always check previous commits how it worked, its not used anymore as (I think) it cannot implement the bridge interface anymore and I may get php stan complaining about unused methods :thinking: